source/eitri/docker_wrapper.py

```diff
@@ -33,28 +33,53 @@
 from retrying import retry
 import psycopg2
 
-# postgres/postgis image
+"""
+This module contains classes about Docker management.
+"""
+
+# Postgres/PostGIS image
 POSTGIS_IMAGE = 'github.com/CanalTP/docker-postgis.git'
 POSTGIS_IMAGE_NAME = 'postgis:2.1'
 
 
 class DbParams(object):
+    """
+    Class to store and manipulate database parameters.
+    """
+
     def __init__(self, host, dbname, user, password):
+        """
+        Constructor of DbParams.
+
+        :param host: the host name
+        :param dbname: the database name
+        :param user: the user name to sue for the database
+        :param password: the password of the user
+        :return:
+        """
         self.host = host
         self.user = user
         self.dbname = dbname
         self.password = password
 
     def cnx_string(self):
+        """
+        The connection string for the database.
+        A string containing all the essentials data for a connection to a database.
+
+        :return: the connection string
+        """
         return 'postgresql://{u}:{pwd}@{h}/{dbname}'.format(
             h=self.host, u=self.user, dbname=self.dbname, pwd=self.password
         )
 
     def old_school_cnx_string(self):
         """
-        old C++ component does not support the classic postgres://...
+        The connection string for the database (old school version).
+        A string containg all the essentials data for a connection to a old school database (before Debian8). Old C++
+        component does not support the classic "postgres://...".
 
-         refactor when moving to debian 8 as the classic cnx string will be recognized
+        :return: the old school connection string
```

Comment:
what is "old school" ?

Reply:
Before Debian8 apparently ^^
You prefere I change this with something more accurate ?